lib/libc/csu/libc_start1.c

```diff
@@ -93,6 +96,11 @@ process_irelocs(void)
 #else
 		crt1_handle_rela(r);
 #endif
+
+#ifdef CRT_IRELOC_CAPRELOC
```

Comment:
NB: This is here, not its own separate case, so both RELA and CAPRELOC can be enabled, envisioning a future where we want to transition from one to the other.